inst/compareScenarios/preprocessing.R

```diff
@@ -18,71 +18,77 @@ options(mip.histRefModel = histRefModel) # nolint
-  "FE|Buildings|Heating|Electricity|Resistance", "GJ/yr", 1e9,
-  "FE|Buildings|Heating|Hydrogen", "GJ/yr", 1e9,
-  "FE|Buildings|Heating|Gases", "GJ/yr", 1e9,
-  "FE|Buildings|Heating|Liquids", "GJ/yr", 1e9,
-  "FE|Buildings|Heating|Solids", "GJ/yr", 1e9,
-  "FE|Buildings|Heating", "GJ/yr", 1e9,
-  "FE|Buildings|Appliances and Light", "GJ/yr", 1e9,
-  "FE|Buildings|Cooking and Water", "GJ/yr", 1e9,
-  "FE|Buildings|Space Cooling", "GJ/yr", 1e9,
-  "FE|Buildings|Space Heating", "GJ/yr", 1e9,
-  "UE|Buildings", "GJ/yr", 1e9,
-  "UE|Buildings|non-Heating|Electricity|Conventional", "GJ/yr", 1e9,
-  "UE|Buildings|Heating|Electricity|Heat pump", "GJ/yr", 1e9,
-  "UE|Buildings|Heating|District Heating", "GJ/yr", 1e9,
-  "UE|Buildings|Heating|Electricity|Resistance", "GJ/yr", 1e9,
-  "UE|Buildings|Heating|Hydrogen", "GJ/yr", 1e9,
-  "UE|Buildings|Heating|Gases", "GJ/yr", 1e9,
-  "UE|Buildings|Heating|Liquids", "GJ/yr", 1e9,
-  "UE|Buildings|Heating|Solids", "GJ/yr", 1e9,
-  "UE|Buildings|Heating", "GJ/yr", 1e9,
-  "ES|Transport|Pass", "km/yr", 1e9,
-  "ES|Transport|Pass|Road|LDV", "km/yr", 1e9,
-  "ES|Transport|Pass|non-LDV", "km/yr", 1e9,
-  "ES|Transport|Pass|Road|LDV|BEV", "km/yr", 1e9,
-  "ES|Transport|Pass|Road|LDV|FCEV", "km/yr", 1e9,
-  "ES|Transport|Pass|Road|LDV|Gases", "km/yr", 1e9,
-  "ES|Transport|Pass|Road|LDV|Hybrid Electric", "km/yr", 1e9,
-  "ES|Transport|Pass|Road|LDV|Liquids", "km/yr", 1e9,
-  "ES|Transport|Freight", "tkm/yr", 1e9,
-  "ES|Transport|Freight|Road|Electric", "tkm/yr", 1e9,
-  "ES|Transport|Freight|Road|FCEV", "tkm/yr", 1e9,
-  "ES|Transport|Freight|Road|Gases", "tkm/yr", 1e9,
-  "ES|Transport|Freight|Road|Liquids", "tkm/yr", 1e9,
-  "Emi|GHG", "t CO2eq/yr", 1e6,
-  "Emi|GHG|Gross|Energy|Supply|Electricity", "t CO2eq/yr", 1e6,
-  "Emi|GHG|Gross|Energy|Supply|Non-electric", "t CO2eq/yr", 1e6,
-  "Emi|GHG|Energy|Demand|Transport", "t CO2eq/yr", 1e6,
-  "Emi|GHG|Energy|Demand|Buildings", "t CO2eq/yr", 1e6,
-  "Emi|GHG|Gross|Energy|Demand|Industry", "t CO2eq/yr", 1e6,
-  "Emi|GHG|Industrial Processes", "t CO2eq/yr", 1e6,
-  "Emi|GHG|Agriculture", "t CO2eq/yr", 1e6,
-  "Emi|GHG|Land-Use Change", "t CO2eq/yr", 1e6,
-  "Emi|GHG|Waste", "t CO2eq/yr", 1e6,
-  "Emi|CO2|CDR|BECCS", "t CO2eq/yr", 1e6,
-  "Emi|CO2|CDR|Industry CCS|Synthetic Fuels", "t CO2eq/yr", 1e6,
-  "Emi|CO2|CDR|DACCS", "t CO2eq/yr", 1e6,
-  "Emi|CO2|CDR|EW", "t CO2eq/yr", 1e6,
-  "Emi|CO2", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Land-Use Change", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Industrial Processes", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Energy|Demand|Transport", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Gross|Energy|Demand|Industry", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Energy|Demand|Buildings", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Gross|Energy|Supply|Non-electric", "t CO2eq/yr", 1e6,
-  "Emi|CO2|Gross|Energy|Supply|Electricity", "t CO2eq/yr", 1e6)
+  "GDP|PPP", "kUS$2005/pCap", 1e6, # creates "GDP|PPP pCap" which is equal to reported variable "GDP|per capita|PPP"
```

Comment:
Probably makes sense to append `pCap` to units in Line 151 instead, just like done for `variable` and `varPlus

Reply:
so just replace `unit = newUnit,`
by
`unit = paste0(newUnit, "/pCap"),` ?

ok, will do so